deploy/chart/local-path-provisioner/values.yaml

```diff
@@ -5,7 +5,7 @@ commonLabels: {}
 
 image:
   repository: rancher/local-path-provisioner
-  tag: master-head
+  tag: v0.0.28
```

Comment:
Can you update it to v0.0.28-dev?

Reply:
At least for me, this tag does not exist:
```
$ docker pull rancher/local-path-provisioner:v0.0.28-dev

What's next:
    View a summary of image vulnerabilities and recommendations → docker scout quickview rancher/local-path-provisioner:v0.0.28-dev
Error response from daemon: manifest for rancher/local-path-provisioner:v0.0.28-dev not found: manifest unknown: manifest unknown
```

Are you sure the `-dev` suffix is correct? I only see an rc1 tag on docker hub.

Reply:
Why not switch to https://github.com/rancher/local-path-provisioner/tree/v0.0.28 for the flux2?

Reply:
I just checked back and you are right, i should switch to just using the tag.

/close